model/sshable.rb

```diff
@@ -100,7 +100,7 @@ def cmd(cmd, stdin: nil, log: true)
   # timeout).
   COMMON_SSH_ARGS = {non_interactive: true, timeout: 10,
                      user_known_hosts_file: [], global_known_hosts_file: [],
-                     keys: [], key_data: [], use_agent: false,
+                     verify_host_key: :accept_new, keys: [], key_data: [], use_agent: false,
```

Comment:
Is this one necessary given how the known hosts are severed from the OS?

Reply:
If OS set this as "yes", ssh agent expects to see the keys as well. Passing empty list means ssh attempt would fail all the time.